app/controllers/api/v1/users_orders_controller.rb

```diff
@@ -3,9 +3,10 @@ class Api::V1::UsersOrdersController < ApplicationController
   
   def create
     order = current_user.orders.create(order_params)
-    params[:items].each do |item|
-      OrderItem.create(order: order, item_id: item.to_i)
+    params[:items].each do |(item, quantity)|
```

Comment:
Had to update how an order was created to grab the item and the quantity and once those were created `order.calc_total` calculates the total for the order. This would be a good spot to abstract some of this behavior away from the controller, perhaps a poro